### PR TITLE
Update the formatting of the PR template

### PR DESCRIPTION
**Pull request by TechStudent11**
### Related Issue/Addition to code
- Updated the formatting of the PR template

#### Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Proposed Changes
- Changed all headers to triple-hashtags.
- Added separator between the screenshots section and the rest of the template.
- Added "Why is this change needed?" field.
- Removed all options from checklist that involved unit testing.

### Why is this change needed?
I think it's needed because the original seemed a bit too big with the unnecessary headers. 

### Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings

---

### Screenshots

Original | Updated
:----------------------:|:-----------:
image  | image

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -13,26 +13,29 @@
...
 - [ ] This change requires a documentation update
 
-# Proposed Changes
+### Proposed Changes
 - change 1
 - change 2
 
-# Additional Info
+### Why is this change needed?
+It's needed because...
```
Comment (KendallDoesCoding) [inline .github/pull_request_template.md:21]:
```suggestion
<!-- How will this change benefit YT-Spammer-Purge? --> 
It's needed because...
```

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -13,26 +13,30 @@
...
+<!-- How will this change benefit YT-Spammer-Purge? --> 
+It's needed because...
+
+### Additional Info
 - any additional info for context
 
-## Checklist:
+### Checklist:
 
 - [ ] My code follows the style guidelines of this project and have read CONTRIBUTING.md
```
Comment (KendallDoesCoding) [inline .github/pull_request_template.md:29]:
```suggestion
- [ ] My code follows the style guidelines of this project and have read [CONTRIBUTING.md](/CONTRIBUTING.md)
```

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
Link the CONTRIBUTING.md file on text CONTRIBUTING.md in the Checklist section.

Preview: https://github.com/KendallDoesCoding/YouTube-Spammer-Purge/blob/patch-18/.github/pull_request_template.md

File: .github/pull_request_template.md (hunk 2 above)
Comment (KendallDoesCoding) [inline .github/pull_request_template.md:29]:
```suggestion
- [ ] My code follows the style guidelines of this project and I have read CONTRIBUTING.md
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
-# Related Issue/Addition to code
+### Related Issue/Addition to code
 <!-- Please delete options that are not relevant. -->
 
 - Fixes #
 - Issue/Addition to code, goes here.
```
Comment (KendallDoesCoding) [inline .github/pull_request_template.md:5]: Is the goes here, really needed? Won't Issue/Addition to code be enough too? What do you think.

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 <!-- Please delete options that are not relevant. -->
 
 - Fixes #<!--issue-no-->
-- Issue/Addition to code, goes here.
+- Addition to code.
```
Comment (KendallDoesCoding) [inline .github/pull_request_template.md:5]: I think Issue should be inculded in this line too, just incase the issue isn't really made as a issue in the repo if you get what I mean?

Comment (TechStudent11) [reply .github/pull_request_template.md:5]: I think it's fine. Someone can add the issue to `Fixes #` if they really want to.

Comment (KendallDoesCoding) [reply .github/pull_request_template.md:5]: Yeah, fair enough.

**Review by EthanHindmarsh — APPROVED** [review]
LGTM ✅
I hope that the `I have added tests` option gets readded soon😄

**Comment by KendallDoesCoding** [comment]
> I hope that the `I have added tests` option gets readded soon😄

cc: @TechStudent11

**Comment by TechStudent11** [comment]
> > I hope that the `I have added tests` option gets readded soon😄
> 
> cc: @TechStudent11

I have notifications on so I know when I get a comment. The unit test options will be added back **when unit testing actually gets added to the program**.

**Review by KendallDoesCoding — APPROVED** [review]

**Pull request merged**